src/edutap/wallet_google/modelbase.py

```diff
@@ -1,43 +1,143 @@
+from .modelcore import GoogleWalletWithIdModel
+from .models.primitives import CallbackOptions
+from .models.primitives import GroupingInfo
+from .models.primitives import Image
+from .models.primitives import PassConstraints
+from .models.primitives import SecurityAnimation
+from .models.primitives.barcode import Barcode
+from .models.primitives.barcode import RotatingBarcode
+from .models.primitives.class_template_info import ClassTemplateInfo
+from .models.primitives.data import ImageModuleData
+from .models.primitives.data import InfoModuleData
+from .models.primitives.data import LinksModuleData
+from .models.primitives.data import TextModuleData
+from .models.primitives.enums import MultipleDevicesAndHoldersAllowedStatus
+from .models.primitives.enums import ViewUnlockRequirement
+from .models.primitives.message import Message
 from pydantic import BaseModel
-from pydantic import ConfigDict
+from pydantic import Field
 
 
-class GoogleWalletModel(BaseModel):
+class GoogleWalletClassModel(GoogleWalletWithIdModel):
     """
-    Base model for all Google Wallet models.
+    Base model for all Google Wallet Class models.
     """
 
-    model_config = ConfigDict(
-        extra="forbid",
-        use_enum_values=True,
```

Comment:
Why dropping this?

Reply:
I moved that to the absolute BaseModel in modelcore.py. 
We could duplicate it in several potential BaseModels, but as all inherit from our GoogleWalletModel BaseModel that shouldn't be necessary.